lib/runner/extensions/event.command.js

```diff
@@ -328,9 +329,16 @@ module.exports = {
                     // @todo - should this be handled by the sandbox?
                     result && result.environment && (result.environment = new sdk.VariableScope(result.environment));
                     result && result.globals && (result.globals = new sdk.VariableScope(result.globals));
+                    result && result.collectionVariables &&
+                        (result.collectionVariables = new sdk.VariableScope(result.collectionVariables));
```

Comment:
@shamasis we are bubbling `_variables` up with the results on the `test` callback. I'm guessing we are okay with this?